### PR TITLE
templating getRadioFields(), getDropDown() in Util.php

### LINKED ISSUE #12006: Templating getRadioFields() and getDropdown() in libraries/Util.php
In libraires/Util.php, make templates from getRadioFields and getDropdown functions which returns HTML and are used in various scripts and other templates as well.
See https://wiki.phpmyadmin.net/pma/Templates
Also, here is an example commit doing similar templating: https://github.com/phpmyadmin/phpmyadmin/commit/39761819c31b89c93bd312a9454178f9f9abb041

### PR DESCRIPTION
**Pull request by poush**
Fixes #12006 
Updated all tests code to pass them on new changes.
Signed-off-by: Piyush Agrawal <poush12@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by poush** [comment]
@nijel @ibennetch Please look into this.

File: libraries/Util.php [hunk 1]
```diff
@@ -2604,40 +2604,49 @@ public static function getRadioFields(
...
-            }
-            $radio_html .= "\n";
+            // $radio_html .= '<input type="radio" name="' . $html_field_name . '" id="'
+                        // . $html_field_id . '" value="'
+                        // . htmlspecialchars($choice_value) . '"';
+
+            if ($choice_value == $checked_choice)
+                $checked = 1;
+            else
+                $checked = 0;
```
Comment (nijel) [inline libraries/Util.php]: Please use {} parenthesis in if statement.

File: libraries/Util.php [hunk 2]
```diff
@@ -2604,40 +2604,49 @@ public static function getRadioFields(
...
+            //             . '</label>';
+
+            // if ($line_break) {
+            //     $radio_html .= '<br />';
+            // }
+
+            // if (! empty($class)) {
+            //     $radio_html .= '</div>';
+            // }
+            // $radio_html .= "\n";
```
Comment (nijel) [inline libraries/Util.php]: Please remove commented out code.

**Review by nijel — CHANGES_REQUESTED** [review]

**Comment by poush** [comment]
Thanks @nijel, updated my PR.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**